reactive/cwr.py

```diff
@@ -72,12 +72,14 @@ def install_juju():
     # matrix not released to pypi yet
     pip_install_from_git(
         3, True, 'https://github.com/juju-solutions/matrix')
-    # juju plugins (for crashdump)
-    utils.run_as('root', 'git', 'clone', 'https://github.com/juju/plugins',
-                 '/usr/local/lib/juju-plugins')
-    for plugin in Path('/usr/local/lib/juju-plugins').iterdir():
-        if plugin.name.startswith('juju-'):
-            (Path('/usr/local/bin') / plugin.name).symlink_to(plugin)
+    # juju plugins (for crashdump). Skip if already deployed.
+    plugins_dir = '/usr/local/lib/juju-plugins'
+    if not os.path.isdir(plugins_dir):
```

Comment:
Be nice if we did a `git pull` if the `plugins_dir` path already exists.  That would ensure the user got the latest from the plugins repo.

If we did this, you'd probably want to move the `for plugin in ... if startswith(juju) ... symlink ...` logic out of the conditional and include an `if not Path(plugin).exists()` before creating a new symlink in `/usr/local/bin`.